### PR TITLE
Set animated background for home page

### PR DESCRIPTION
**Pull request by GusMeg**
## Description

Set animated background for home page. Included module tsparticles for rendering small moving ethers in the background of the home page.

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #6 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: gusmeg.eth

https://github.com/BuidlGuidl/batch1.buidlguidl.com/assets/104281578/c8bccf94-8477-4181-8548-db7c61fdb1cb

### CODE REVIEW

File: package-lock.json
Comment (technophile-04) [inline package-lock.json:1]: Remove this, it seems that you have used `npm install` instead of `yarn add`

File: package.json [hunk 1]
```diff
@@ -32,10 +32,18 @@
...
   },
   "resolutions": {
     "usehooks-ts@^2.7.2": "patch:usehooks-ts@npm:^2.7.2#./.yarn/patches/usehooks-ts-npm-2.7.2-fceffe0e43.patch"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-tsparticles": "^2.12.2",
+    "tsparticles": "^3.1.0",
+    "tsparticles-engine": "^2.12.0",
+    "tsparticles-slim": "^2.12.0"
```
Comment (technophile-04) [inline package.json:47]: This is root `package.json`, I think you bymistake added all depeendencies here. 

Make sure you first `cd packages/nextjs` and then `yarn add` all the dependencies

File: .editorconfig
Comment (technophile-04) [inline .editorconfig:1]: Remove this, we already have prettier configured 🙌

Comment (GusMeg) [reply .editorconfig:1]: Removed!!

File: package.json (hunk 1 above)
Comment (GusMeg) [reply package.json]: Whoops! Thanks for spotting that. Corrected !

File: package-lock.json
Comment (GusMeg) [reply package-lock.json:1]: Removed!!

File: .editorconfig
Comment (technophile-04) [inline .editorconfig:1]: We still have this, lets remove 🙌

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/react": "^18.0.9",
```
Comment (technophile-04) [inline package.json:35]: Lets remove this from here too we already have `@types/react` in nextjs

You could run `yarn remove @types/react` and it should remove it from root

Comment (GusMeg) [reply package.json]: Forgot to remove that one, sorry!! I updated the file and removed that line.

File: .editorconfig
Comment (GusMeg) [reply .editorconfig:1]: Sorry!! I just removed it 👍

**Review by technophile-04 — APPROVED** [review]
Just pushed a small commit removing the callback function `particlesLoaded`  since we not doing anything on particles loaded.

Also just wanted to note that [`react-tsparticles`](https://www.npmjs.com/package/react-tsparticles) is been deprecated and they suggest using [`@tsparticles/react`](https://github.com/tsparticles/react), since it working fine for us and we are not using it heavily it's fine but in future we should try avoiding deprecated packages 🙌


Thanks @GusMeg this is looking great !!

**Pull request merged**